Cargo.toml

```diff
@@ -19,7 +19,7 @@ xml-rs = "0.8.4"
 strum = { version = "0.22.0", features = ["derive"] }
 
 [target.'cfg(target_env = "msvc")'.dependencies.windows]
-version = "0.24.0"
+version = "0.34.0"
```

Comment:
The version under `[target.'cfg(target_env = "gnu")'.dependencies.windows]` also needs to be updated to 0.34.0